WalletWasabi/JsonConverters/SmartLabelJsonConverter.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.JsonConverters
+{
+	public class SmartLabelJsonConverter : JsonConverter
+	{
+		public override bool CanConvert(Type objectType)
+		{
+			return objectType == typeof(SmartLabel);
+		}
+
+		/// <inheritdoc />
+		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+		{
+			var s = reader.Value as string;
+			return new SmartLabel(s);
+		}
+
+		/// <inheritdoc />
+		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
```

Comment:
Are there any tests to write and read back a SmartLabel using this?

Reply:
Yes, there are jsonserializing tests.